service/src/test/java/uk/gov/hmcts/reform/fpl/model/CaseDataTest.java

```diff
@@ -247,4 +284,205 @@ void shouldReturnNullIfC2DocumentBundleIsNotPopulated() {
             assertThat(caseData.getLastC2DocumentBundle()).isEqualTo(null);
         }
     }
+
+    @Nested
+    class PrepareCaseManagementOrder {
+        final Schedule schedule = Schedule.builder().includeSchedule("Yes").build();
+        final List<Element<Recital>> recitals = wrapElements(Recital.builder().title("example title").build());
+        final OrderAction action = baseOrderActionWithType().build();
+
+        @Test
+        void shouldReturnCaseManagementOrderWhenFullDetailsButNoPreviousOrder() {
+            CaseData caseData = getCaseData();
+
+            assertThat(caseData.getCaseManagementOrder())
+                .isEqualToComparingFieldByField(buildOrder(schedule, recitals, createCmoDirections(), action));
+        }
+
+        @Test
+        @DisplayName("can pull directions from order in about to start callback")
```

Comment:
I am not sure if it's worth mentioning anything related to about to start here, should the case data has this knowledge? It seems like it is wrong direction.

Reply:
Maybe the test should be somewhere else then. (could be related to to my controller question).

Maybe we can have a controller unit test?

Reply:
I think I was more just trying to explain the use cases for the tests, maybe not necessary at all? Happy to remove the display names